README.md

```diff
@@ -1,9 +1,25 @@
-# openinput-cm3
+# openinput-mouse
 
-Open source mouse firmware based on [libopencm3](https://github.com/libopencm3/libopencm3).
+Open source mouse firmware.
+
+```
+.
+├── tools	# Utilities
+├── Targets	# Meson cross-compile files
+├── src		# Source files
+│   ├── drv  # Drivers, for mcus and other devices like sensors
+│   │    ├── mcu  # Drivers for family of mcu
+│   │    ├── sensor  # Drivers for optical mouse sensors
+│   │    └── ...  # Other driver classes
+│   └── target	# Target abstraction
+│         ├── rival310 # Target specific to rival310
+│         └── stm32f1 # Generic stm32f1 family target
+└── external  # Sources from external projects like tinyusb
+```
 
 ### Supported Devices
-- Blue pill
+- stm32f1 family
```

Comment:
Hum, I think this would be better in a new section, `Supported MCU Families`. And use this notation: `STM32F1 Family`.

Reply:
I agree.